src/libraries/System.Net.Requests/src/System/Net/WebRequest.cs

```diff
@@ -385,15 +383,15 @@ internal static List<WebRequestPrefixElement> PrefixList
             }
         }
 
-        public static RequestCachePolicy DefaultCachePolicy { get; set; } = new RequestCachePolicy(RequestCacheLevel.BypassCache);
+        public static RequestCachePolicy? DefaultCachePolicy { get; set; } = new RequestCachePolicy(RequestCacheLevel.BypassCache);
```

Comment:
Not sure why this need to nullable

Reply:
Simply because .NET Framework allows you to set it to null.